assets/js/base/context/providers/cart-checkout/checkout/processor/index.js

```diff
@@ -111,12 +111,7 @@ const CheckoutProcessor = () => {
 
 	const checkValidation = useCallback( () => {
 		if ( hasValidationErrors ) {
-			return {
```

Comment:
I don't think this is necessary since inputs show inline validation notices. Agree?

Reply:
I agree that the notice feels redundant. I believe design added it. We can check later.